migrate2rocky/migrate2rocky.sh

```diff
@@ -1185,7 +1185,7 @@ fi
 if [[ $verify_all_rpms && $convert_to_rocky ]]; then
   generate_rpm_info finish
   infomsg $'You may review the following files:\n'
-  find "$convert_info_dir" -type f -name "$HOSTNAME-rpms-*.log"
+  printf '%s\n' "$convert_info_dir"/$HOSTNAME-rpm-list-*.log
```

Comment:
$HOSTNAME needs to be quoted here:
`"$convert_info_dir/$HOSTNAME-rpm-list-"*.log`

Reply:
Good catch, thanks!

I was forgetting that it's only in my little world that hostname's don't contain shell metacharacters.  That's definitely not a good thing to assume in a script like this.  :)

Reply:
Looks good now, thanks